package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rn-base-project",
-  "version": "0.0.3",
+  "version": "0.0.5",
   "main": "index.js",
   "repository": "https://github.com/saigon-technology/rn-base-project.git",
   "author": "React Native Teeam <SaiGon Technology Solutions>",
```

Comment:
Why you update this file. please rebase from branch `develop` or find root cause for this changed

Reply:
I fixed it to the original version

Reply:
Could you make sure you get the latest code.